Infrastructure/Pages/HomePage.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium;
+
+namespace Infrastructure.Pages
+{
+    public class HomePage : BasePage
```

Comment:
This class doesn't have any properties besides representing the home page, perhaps you can add things like collecting the products in a list which then you can take specific items from and add them to the cart, etc.